aws/resource_aws_ssm_maintenance_window.go

```diff
@@ -51,6 +51,21 @@ func resourceAwsSsmMaintenanceWindow() *schema.Resource {
 				Optional: true,
 				Default:  true,
 			},
+
+			"end_date": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
do you want some validation on these checking the date format?

Reply:
The API parameter says it accepts ISO-8601 extended format, which has some slight differences with RFC3339, that would match our normal date validators. The `time.RFC3339` format built into Go's standard library doesn't work with all ISO-8601 formats unfortunately.

It looks like we have ISO-8601 needs in other places where it is inconsistently handled, so I have created a technical debt issue to address all these at once (I would think preferably upstream in the provider SDK), but even a common solution in the provider itself is a good start:  https://github.com/terraform-providers/terraform-provider-aws/issues/7045